README.md

```diff
@@ -1,117 +1,32 @@
-# go-policy-template
+# Namespace Label Propagator
 
-This is a template repository that can be used to quickly scaffold a
-Kubewarden policy written with Go language.
+This policy ensures that all workloads deployed in the cluster contain specific
+labels defined in the namespace where the workload will be executed.
 
-Don't forget to checkout Kubewarden's [official documentation](https://docs.kubewarden.io)
-for more information about writing policies.
+When a resource is created or updated, this policy copies a list of user-specified
+labels from the namespace to the object. Labels defined on the namespace take
+precedence over labels already defined inside the resource.
 
-## Introduction
+This policy is able to set the labels for the following resource kinds: `Pod`,
+`ReplicationController`, `Deployment`, `ReplicaSet`, `StatefulSet`, `DaemonSet`, `Job` and `CronJob`.
```

Comment:
Documenting the new feature that you implemented:

```suggestion
This policy is able to set the labels for the following resource kinds: `Pod`,
`ReplicationController`, `Deployment`, `ReplicaSet`, `StatefulSet`, `DaemonSet`, `Job` and `CronJob`.

When dealing with Kubernetes resources that generate pods, the policy ensures the special labels
are propagated also to them.
```

Once merged, don't forget to update the artifact hub metadata.